cfme/infrastructure/host.py

```diff
@@ -67,6 +67,7 @@
     }
 )
 
+# REVIEW: Need to find one of these, would love to have a div#id parent
 drift_table = CheckboxTable("//table[@class='style3']")
```

Comment:
@jkrocil (or anyone) Do you have an appliance handy that already has drift results, so we can fix this locator up? Even better would be two appliances, a 53z and a 54z or newer.